data/collections.yaml

```diff
@@ -3,7 +3,13 @@ collections:
       slug: how-to-use-search-patterns-on-sourcegraph
       type: posts
       members:
-        - three-ways-to-search-code-with-sourcegraph
         - how-to-search-code-with-sourcegraph-using-literal-patterns
         - how-to-search-with-sourcegraph-using-regular-expression-patterns
         - how-to-search-with-sourcegraph-using-structural-patterns
+    - title: Understanding Git Basics
```

Comment:
Open to discussion on this! Was using it mainly to test out behavior

Reply:
Yeah, I think this makes sense, I was thinking of adding more too.